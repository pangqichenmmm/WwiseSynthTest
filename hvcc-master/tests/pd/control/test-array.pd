#N canvas 0 22 498 609 10;
#N canvas 0 22 450 278 (subpatch) 0;
#X array array1 100 float 3;
#A 0 0.0142857 -0.528575 -0.519052 -0.509528 -0.500004 -0.483337 -0.46667
-0.450004 -0.433337 -0.41667 -0.400003 -0.383932 -0.36786 -0.351789
-0.335717 -0.319646 -0.303574 -0.287502 -0.271431 -0.257145 -0.242859
-0.228574 -0.214288 -0.200002 -0.185716 -0.17143 -0.157145 -0.142859
-0.128573 -0.112501 -0.0964299 -0.0803584 -0.0642868 -0.0482153 -0.0321438
-0.0160722 -7.15256e-07 0.0142851 0.0285709 0.0428567 0.0571425 0.0714283
0.471431 0.414288 0.385716 0.342859 0.314287 0.300001 0.285715 0.257144
0.228572 0.214286 -0.733338 -0.800005 0.871434 0.871434 0.871434 0.871434
0.854291 0.837148 0.820005 0.802862 0.785719 0.760719 0.735719 0.710719
0.685719 0.666671 0.647623 0.628575 0.610718 -1.13249e-06 -0.171431
-0.221431 -0.271431 -0.304765 -0.338098 -0.371432 -0.404766 -0.438099
-0.471433 -0.480956 -0.49048 -0.500004 -0.500004 -0.500004 -0.500004
-0.500004 -0.500004 -0.500004 -0.500004 -0.500004 -0.500004 -0.492861
-0.485718 -0.478576 -0.471433 -0.447623 -0.423813 -0.171431;
#X coords 0 1 99 -1 200 140 1;
#X restore 126 -103 graph;
#N canvas 0 22 450 278 (subpatch) 0;
#X array \$0-array2 100 float 3;
#A 0 0.371425 0.328571 0.3 0.242858 0.014289 -0.0142821 -0.114281 -0.149995
-0.185709 -0.228566 -0.264279 -0.357136 -0.414278 -0.47142 -0.499991
-0.528562 -0.542848 -0.557134 -0.585705 -0.59999 -0.59999 -0.614276
-0.628561 -0.628561 -0.628561 -0.628561 -0.628561 -0.628561 -0.628561
-0.628561 -0.628561 -0.585705 -0.557134 -0.528562 -0.492849 -0.457135
-0.385707 -0.314279 -0.242851 -0.178566 -0.114281 -0.049996 0.014289
0.0857168 0.157145 0.285715 0.371428 0.457142 0.599997 0.628568 0.657139
0.714282 0.757138 0.78571 0.814281 0.835709 0.857137 0.871423 0.885708
0.899994 0.91428 0.921422 0.928565 0.935708 0.942851 0.942851 0.942851
0.942851 0.942851 0.935708 0.928565 0.91428 0.899994 0.878566 0.857137
0.814281 0.757138 0.721424 0.685711 0.614283 0.442856 0.399999 0.357143
0.257144 0.17143 0.0714313 3.45707e-06 -0.128567 -0.199994 -0.257137
-0.299993 -0.335707 -0.371421 -0.385707 -0.399992 -0.414278 -0.428563
-0.428563 -0.428563 -0.34285;
#X coords 0 1 100 -1 200 140 1;
#X restore 116 73 graph;
#X obj 193 347 tabread \$0-array2;
#X obj 154 274 t b;
#X obj 154 298 f;
#X obj 185 297 + 1;
#X obj 154 247 loadbang;
#X obj 154 320 sel 100;
#X obj 193 369 t f b;
#X obj 193 391 print;
#X connect 2 0 8 0;
#X connect 3 0 4 0;
#X connect 4 0 5 0;
#X connect 4 0 7 0;
#X connect 5 0 4 1;
#X connect 6 0 3 0;
#X connect 7 1 2 0;
#X connect 8 0 9 0;
#X connect 8 1 3 0;
