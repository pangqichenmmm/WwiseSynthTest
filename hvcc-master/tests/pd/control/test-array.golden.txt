[@ 0.000] print: -0.34285
[@ 0.000] print: -0.428563
[@ 0.000] print: -0.428563
[@ 0.000] print: -0.428563
[@ 0.000] print: -0.414278
[@ 0.000] print: -0.399992
[@ 0.000] print: -0.385707
[@ 0.000] print: -0.371421
[@ 0.000] print: -0.335707
[@ 0.000] print: -0.299993
[@ 0.000] print: -0.257137
[@ 0.000] print: -0.199994
[@ 0.000] print: -0.128567
[@ 0.000] print: 3.45707e-06
[@ 0.000] print: 0.0714313
[@ 0.000] print: 0.17143
[@ 0.000] print: 0.257144
[@ 0.000] print: 0.357143
[@ 0.000] print: 0.399999
[@ 0.000] print: 0.442856
[@ 0.000] print: 0.614283
[@ 0.000] print: 0.685711
[@ 0.000] print: 0.721424
[@ 0.000] print: 0.757138
[@ 0.000] print: 0.814281
[@ 0.000] print: 0.857137
[@ 0.000] print: 0.878566
[@ 0.000] print: 0.899994
[@ 0.000] print: 0.91428
[@ 0.000] print: 0.928565
[@ 0.000] print: 0.935708
[@ 0.000] print: 0.942851
[@ 0.000] print: 0.942851
[@ 0.000] print: 0.942851
[@ 0.000] print: 0.942851
[@ 0.000] print: 0.942851
[@ 0.000] print: 0.935708
[@ 0.000] print: 0.928565
[@ 0.000] print: 0.921422
[@ 0.000] print: 0.91428
[@ 0.000] print: 0.899994
[@ 0.000] print: 0.885708
[@ 0.000] print: 0.871423
[@ 0.000] print: 0.857137
[@ 0.000] print: 0.835709
[@ 0.000] print: 0.814281
[@ 0.000] print: 0.78571
[@ 0.000] print: 0.757138
[@ 0.000] print: 0.714282
[@ 0.000] print: 0.657139
[@ 0.000] print: 0.628568
[@ 0.000] print: 0.599997
[@ 0.000] print: 0.457142
[@ 0.000] print: 0.371428
[@ 0.000] print: 0.285715
[@ 0.000] print: 0.157145
[@ 0.000] print: 0.0857168
[@ 0.000] print: 0.014289
[@ 0.000] print: -0.049996
[@ 0.000] print: -0.114281
[@ 0.000] print: -0.178566
[@ 0.000] print: -0.242851
[@ 0.000] print: -0.314279
[@ 0.000] print: -0.385707
[@ 0.000] print: -0.457135
[@ 0.000] print: -0.492849
[@ 0.000] print: -0.528562
[@ 0.000] print: -0.557134
[@ 0.000] print: -0.585705
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.628561
[@ 0.000] print: -0.614276
[@ 0.000] print: -0.59999
[@ 0.000] print: -0.59999
[@ 0.000] print: -0.585705
[@ 0.000] print: -0.557134
[@ 0.000] print: -0.542848
[@ 0.000] print: -0.528562
[@ 0.000] print: -0.499991
[@ 0.000] print: -0.47142
[@ 0.000] print: -0.414278
[@ 0.000] print: -0.357136
[@ 0.000] print: -0.264279
[@ 0.000] print: -0.228566
[@ 0.000] print: -0.185709
[@ 0.000] print: -0.149995
[@ 0.000] print: -0.114281
[@ 0.000] print: -0.0142821
[@ 0.000] print: 0.014289
[@ 0.000] print: 0.242858
[@ 0.000] print: 0.3
[@ 0.000] print: 0.328571
[@ 0.000] print: 0.371425
